scripts/supervisord/assemble-and-restart

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -x
+set -eo pipefail
+
+# S2i fails when /tmp/src/ is empty. 
+# Here we will create an empty folder / dummy file
+mkdir -p /tmp/src
+touch /tmp/src/.dummy
+
+# HACK. For some reason, *somewhere* $PATH is being overriden and the default
+# "/opt/rh/" is not being added. We instead manually search for the binaries and add it
+# to $PATH.
+# REMOVE THIS BEFORE MERGING.
+for i in $(find /opt/rh/ -name 'bin' -type d | grep '/root/usr/bin'); do
+        PATH=$i:$PATH
+done
+
+# Override PATH for the assemble script so that /usr/bin precedes
+# /opt/app-root/bin so wrong Python version isn't found during setup
+# if using Python S2I base image.
+# as per: https://github.com/openshift-evangelists/pseudo-vps-quickstart/blob/master/.s2i/bin/assemble#L7
+PATH=/usr/bin:$PATH
```

Comment:
We shouldn't need that as we no longer use Python SupervisorD

Reply:
The above comments I added describe why, and that's because of the $PATH issues we were having before. `/usr/bin` must be before everything else in order for the Python S2I to work. I will remove this after I've found the bug.